package.json

```diff
@@ -74,6 +74,7 @@
     "redux-logger": "3.0.6",
     "styled-components": "5.3.0",
     "styled-system": "5.1.5",
+    "type-fest": "^2.12.1",
```

Comment:
We should pin the dependency for security reasons. (if we decide to add)

Reply:
My bad, keep forgetting it. Pinned with https://github.com/yearn/yearn-finance-v3/pull/562/commits/325ce7f07d7cd4803fa8d5a97b4c12aa24e5f39e